test/metabase/sync_database/analyze_test.clj

```diff
@@ -72,3 +78,80 @@
 (expect false (values-are-valid-emails? [100]))
 (expect false (values-are-valid-emails? ["true"]))
 (expect false (values-are-valid-emails? ["false"]))
+
+;; Tests to avoid analyzing hidden tables
+(defn count-un-analyzed-fields [db_id table-name]
+  (let [table-id (db/select-one-id table/Table, :db_id db_id, :active true :name table-name)]
+    (assert (pos? ;; don't let ourselves be fooled if the test passes because the table is
+                  ;; totally broken or has no fields. Make sure we actually test something
+             (db/count metabase.models.field/Field :table_id table-id)))
+    (db/count metabase.models.field/Field :last_analyzed nil :table_id table-id)))
```

Comment:
Basically it's the style guide advice here https://github.com/bbatsov/clojure-style-guide#opt-commas-in-map-literals. That's talking about maps but IMO it applies here too since it's still key-value pairs